browser/ui/sidebar/sidebar_controller.cc

```diff
@@ -96,11 +110,36 @@ void SidebarController::ActivateItemAt(absl::optional<size_t> index,
   LoadAtTab(item.url);
 }
 
+bool SidebarController::ActiveTabFromOtherBrowsersForHost(const GURL& url) {
```

Comment:
Can we use Singleton tab for this? https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/singleton_tabs.h;l=21?q=singletontab&ss=chromium

Reply:
`ShowSingletonTab()` doesn't fit well because it checks with url.
However, we should activate Talk tab by checking host name.